xhtml2pdf/context.py

```diff
@@ -173,7 +173,16 @@ def atFontFace(self, declarations):
         # Font style
         italic = str(data.get("font-style", "")).lower() in ("italic", "oblique")
 
-        src = self.c.getFile(data["src"], relative=self.c.cssParser.rootPath)
+        # The "src" attribute can be a CSS group but in that case
+        # ignore everything except the font URI
+        uri = data['src']
+        if not isinstance(data['src'], str):
+            for part in uri:
+                if isinstance(part, str):
```

Comment:
This is a very un-semantic way of knowing which part of the CSS group that's the URI :) But let's do it :)